modules/caddyhttp/rewrite/caddyfile.go

```diff
@@ -25,20 +27,116 @@ func init() {
 
 // parseCaddyfile sets up the handler from Caddyfile tokens. Syntax:
 //
-//     rewrite [<matcher>] <to>
+//     rewrite [<matcher>] [<to>] {
+//         to                <string>
+//         method            <string>
+//         strip_path_prefix <string>
+//         strip_path_suffix <string>
+//         rehandle          <bool>
+//         http_redirect     <status_code>
```

Comment:
I am wondering if it is useful to expose the http_redirect feature in the Caddyfile...

Reply:
There is already the `redir` directive, so I'm not sure how useful it is to have it here as well.

Reply:
Let's remove it then, if that's OK with you. We'll expose as little as possible until it is needed.

Reply:
Will do.